content/source/docs/enterprise/api/applies.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: enterprise2
+page_title: "Applies - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-applies"
+---
+
+# Applies API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+An apply represents the results of applying a Terraform Run's execution plan.
+
+## Show an apply
+
+`GET /applies/:id`
+
+Parameter | Description
+----------|------------
+`id`      | The ID of the apply to show.
+
+As there is no endpoint to list applies, you can find the ID for an apply in the
+`relationships.apply` property of a run object.
```

Comment:
I might remove "As" here and just say "There is no endpoint to list applies. You can find..."